Orange/widgets/cpewidget.py

```diff
@@ -32,7 +36,11 @@ def sync_properties(self):
         # to the widget
         for n, p in self.node.ports(direction='IN*', editable=True).items():
             if n not in settings or settings[n] is None:
-                super().__setattr__(n, getattr(self.node, n))
+                value = getattr(self.node, n)
+                if (isinstance(p, ListPort) or isinstance(p, IntPort)) and value is None:
```

Comment:
I am sticking to what the Jira item says, so far you only mentioned ListPort, IntPort types